src/site/stages/build/drupal/api.js

```diff
@@ -34,14 +35,29 @@ function getDrupalClient(buildOptions) {
     Authorization: `Basic ${encodedCredentials}`,
     'Content-Type': 'application/json',
   };
+  const agent = new SocksProxyAgent('socks://127.0.0.1:2001');
 
   return {
+    // We have to point to aws urls on Jenkins, so the only
+    // time we'll be using cms.va.gov addresses is locally,
+    // when we need a proxy
+    usingProxy: address.includes('cms.va.gov'),
```

Comment:
Originally I had this checking for a buildtype of localhost, too, but this way we preserve the ability to use different build types locally